utility_analysis_new/combiners.py

```diff
@@ -601,22 +601,28 @@ def __init__(self, params: pipeline_dp.combiners.CombinerParams,
         self._params = params
         self._error_quantiles = error_quantiles
 
-    def create_accumulator(self, prob_to_keep: float) -> AccumulatorType:
+    def create_accumulator(
+            self, prob_to_keep: float) -> PartitionSelectionAccumulator:
         """Creates an accumulator for metrics."""
-        return PartitionSelectionAccumulator(probabilities=[prob_to_keep])
+        return ((prob_to_keep,), None)
```

Comment:
What does the `,` after `prob_to_keep` do?

(Also exists in some places in the tests)

Reply:
`(10)` is an expression which equals `10` (just by arithmetic rules, as in any programming language)
`(10,)` is a tuple with 1 element `10`